imports/plugins/core/accounts/server/methods/sendResetPasswordEmail.js

```diff
@@ -47,9 +47,7 @@ async function sendResetEmail(userId, optionalEmail, language) {
   }
 
   // Create token for password reset
-  const token = Random.secret();
-  const when = new Date();
-  const tokenObj = { token, email, when };
+  const tokenObj = generateVerificationTokenObject({ email });
```

Comment:
This file has already been refactored in another PR that isn't merged yet. Will update that PR with this new `generateVerificatinTokenObject` once this PR is merged.

Reply:
@kieckhafer There were conflicts when I merged that PR and I must have resolved them incorrectly. Make sure any changes from #5556 are in your non-Meteor version of the function, but otherwise OK to re-delete.